src/Sfa.Tl.ResultsAndCertification.Web/Mapper/IndustryPlacementMapper.cs

```diff
@@ -38,6 +38,7 @@ public IndustryPlacementMapper()
                .ForMember(d => d.RegistrationPathwayId, opts => opts.MapFrom(s => s.RegistrationPathwayId))
                .ForMember(d => d.PathwayId, opts => opts.MapFrom(s => s.TlPathwayId))
                .ForMember(d => d.AcademicYear, opts => opts.MapFrom(s => s.AcademicYear))
+               .ForMember(d => d.CompletionYear, opts => opts.MapFrom(s => s.AcademicYear + 2))
```

Comment:
Just a suggestion probably we could say here CompletionAcademicYear rather than CompletionYear because it would define the correct meaning of the resource.